packages/doltgres/content/reference/sql/server/metrics.md

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
I don't think this is true yet.

Reply:
I think it works, I don't know why it wouldn't